cellbase-lib/src/main/java/org/opencb/cellbase/lib/impl/core/OntologyMongoDBAdaptor.java

```diff
@@ -64,6 +64,8 @@ public CellBaseIterator<OntologyTerm> iterator(OntologyQuery query) throws CellB
         QueryOptions queryOptions = query.toQueryOptions();
 
         MongoDBCollection mongoDBCollection = getCollectionByRelease(mongoDBCollectionByRelease, query.getDataRelease());
+        System.out.println("ontology mongodb adaptor: mongoDBCollection = " + mongoDBCollection.toString() + ", data release = "
```

Comment:
Wouldn't it be better to use logger.info instead?

Reply:
oopppss, it's to removed !